resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -668,4 +668,8 @@ settings.regions.none_selected=No region selected
 settings.regions.recent=Recent Regions
 settings.regions.region_sub_menu=All Regions
 settings.title=AWS Connection Settings
+sqs.queue.message.sampling=Message Sampling
```

Comment:
is "Sampling" the same word use by SQS in the web console? I wonder if there is a more common name for this.

Reply:
It is not used in the web console - I added "sampling" to make it clear that the user can view only a sample of the messages in the queue. Perhaps "Messages received" or "Polled messages" would be more familiar.

Reply:
How about the wording used on the console "Poll for Messages"

Reply:
Makes sense,  the idea is to convey that this is a (partial) snapshot at an instant in time. I wonder if there is precedent in tools like WireShark. Or CloudWatch...

I don't have much better suggestions: "snapshot", "glance". Alterntiavely,  UIs like CloudWatch tend to explicitly say actual numbers: something like `X (out of Y) messages polled at 05:42`.

But here I see you just need a reasonable title for the editor tab. `Message Sample` instead of `Message Sampling` might be slightly more familiar IMO. Or if the API supports it, consider `Messages (4 of 73 @ 05:52)`.

Reply:
> How about the wording used on the console "Poll for Messages"

For the tab title this could be `Polled Messages`.

Reply:
I think changing the name to what console uses would work. In the future, I plan on adding a warning dialog about polling messages whenever the user opens the queue. I can clarify that the messages are only a snapshot in the dialog.